addons/binding/org.openhab.binding.solaredge/ESH-INF/thing/generic-channel-types.xml

```diff
@@ -0,0 +1,147 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="solaredge" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+	<channel-type id="live-type-production">
+		<item-type>Number</item-type>
+		<label>Production</label>
+		<description>Current Production</description>
+		<state pattern="%.2f kW" readOnly="true">
```

Comment:
Would it make sense to integrate UoM

https://www.eclipse.org/smarthome/documentation/concepts/units-of-measurement.html

Reply:
Is there any other unit than "kW" people would like to use? Of course you could use kJ/h instead but I think this is pointless...

Reply:
Is there any further documentation on how to use UoM as developer. The link above explains more or less what is changed from user's perspective.

If I simply change the type to "Number:Energy" I get this error:
Exception while formatting value '12.48' of item SE_DAY_PRODUCTION with format '%.1f %unit%': Conversion = 'u'
I did not expect this to be working fine but I hoped to get a more useful hint on what to adopt.

Reply:
Have a look at https://www.eclipse.org/smarthome/blog/2018/02/22/units-of-measurement.html, there is some more info. Also check e.g. the code of the Yahoo Weather binding as an example.